src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -106,38 +120,69 @@ class Liberty implements Plugin<Project> {
             logging.level = LogLevel.INFO
         }
 
-        project.task('libertyDebug', type: DebugTask, dependsOn: 'libertyCreate') {
+        project.task('libertyDebug', type: DebugTask) {
             description 'Runs the Liberty Profile server in the console foreground after a debugger connects to the debug port (default: 7777).'
             logging.level = LogLevel.INFO
+            
+            project.afterEvaluate {
+                dependsOn installAppsDependsOn(server, 'installLiberty')
+            }
```

Comment:
I am questioning this one a bit just because I imaging that doing a libertyDebug is something they would do after say fail integration test or just general debugging on a set up server.  So, I am sort of thinking we might not want to dependOn any task for debug and just make them do the setup separately.  Otherwise, we could be updating the server files. Get @mattbsox and @jjvilleg opinion too.

Reply:
done